crates/aptos/src/move_tool/fmt.rs

```diff
@@ -127,31 +127,39 @@ impl FmtCommand {
             for file in &files {
                 if !file.exists() {
                     return Err(CliError::UnexpectedError(format!(
-                        "File does not exist:{:?}",
-                        file
+                        "File does not exist:{}",
+                        file.display()
                     )));
                 }
                 if !file.is_file() {
                     return Err(CliError::UnexpectedError(format!(
-                        "Path is not a file:{:?}",
-                        file
+                        "Path is not a file:{}",
+                        file.display()
                     )));
                 }
                 if file.extension().unwrap() != move_command_line_common::files::MOVE_EXTENSION {
                     return Err(CliError::UnexpectedError(format!(
-                        "File is not a Move file:{:?}",
-                        file
+                        "File is not a Move file:{}",
+                        file.display()
                     )));
                 }
             }
             files
         } else {
             // Handle package path
-            let package_opt = if let Some(path) = package_opt {
-                fs::canonicalize(path.as_path()).ok()
-            } else {
-                None
+            let package_opt = match package_opt {
```

Comment:
This was intended before adding support of formatting files but I guess it makes more sense to ask for explicit path. Also, please update `CHANGELOG.md` for CLI under `Unreleased` section with this new feature.

Reply:
Are we sure this was intended? Note that in the initial version (before adding support for file formatting), if a user calls `aptos move fmt --package-path /Foo/Bar/Package` from the current `/Alpha/Beta/Package` and the path `/Foo/Bar/Package` does not exist on the users file system, then the directory `/Alpha/Beta/Package` gets formatted. If this is the intended behavior, I can squish these into single PR. But I figured this is not intended behavior which is why I separated.

Reply:
I will make a PR for CHANGELOG.md

Was unaware of this file - thanks for notifying.

Reply:
you are right, it is indeed a bug. Thanks for fixing it.

Reply:
I didn't realize `Changelog.md` is also in this repo. I added the changes to it (for both prs) directly to this pr.